mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1157,11 +1157,11 @@
     <!-- Name of the marketing notification channel. Displayed in the "App notifications" system settings for the app -->
     <string name="notification_marketing_channel_name">Marketing</string>
     <!-- Title shown in the notification that pops up to remind the user to set fenix as default browser.
-    %1$s is a placeholder that will be replaced by the app name (Fenix). -->
-    <string name="notification_default_browser_title">%1$s is fast and private</string>
+    The app name is in the text, due to limitations with localizing Nimbus experiments -->
```

Comment:
@jhugman : hey - I think you committed these two strings that came to me in this PR, from what I was able to track from commit history.
I didn't spot this right away so accidentally merged this PR - but both these strings need updated string IDs, otherwise, localizers will not see they have changed in Pontoon, and Firefox will remain a placeholder in almost all cases. Thanks!